language/stdlib/modules/libra.move

```diff
@@ -138,6 +138,11 @@ module Libra {
         MintCapability<CoinType> { }
     }
 
+    public fun grant_mint_capability_for_sender<CoinType>() {
+        Transaction::assert(Transaction::sender() == 0xB1E55ED, 0);
```

Comment:
You probably want an additional assertion that `CoinType` is a registered currency here as well (`assert_is_coin`). I also just realized looking at this, that we also want to add this to `grant_burn_capability_for_sender` if you could add this same check to that as well.